tests/api/RestfulServiceTest.php

```diff
@@ -196,38 +196,11 @@ public function testHttpErrorWithoutCache() {
 	public function testHttpErrorWithCache() {
 		$subUrl = 'RestfulServiceTest_Controller/httpErrorWithCache';
 		$connection = new RestfulServiceTest_MockErrorService(Director::absoluteBaseURL(), 0);
-		$this->createFakeCachedResponse($connection, $subUrl);
 		$response = $connection->request($subUrl);
 		$this->assertEquals(400, $response->getStatusCode());
-		$this->assertEquals("Cache response body",$response->getCachedBody());
+		$this->assertEquals(false,$response->getCachedBody());
```

Comment:
This should use `assertFalse` but, more importantly, this now no longer tests the actual functionality it was intended to test.